Library/Formula/when.rb

```diff
@@ -1,12 +1,15 @@
-require 'formula'
-
 class When < Formula
-  homepage 'http://www.lightandmatter.com/when/when.html'
-  url 'http://ftp.de.debian.org/debian/pool/main/w/when/when_1.1.33.orig.tar.gz'
-  sha1 'e314a64d74f79be07da1ade6d39a1fc51205f81d'
+  homepage "http://www.lightandmatter.com/when/when.html"
+  url "http://ftp.de.debian.org/debian/pool/main/w/when/when_1.1.34.orig.tar.gz"
+  mirror "https://mirrors.ocf.berkeley.edu/debian/pool/main/w/when/when_1.1.34.orig.tar.gz"
+  sha256 "ce0540bde96b361d6d0770803901364a687d971ffedd33e36f8f8bef32b19600"
+  head "https://github.com/bcrowell/when.git"
 
   def install
-    bin.install 'when'
-    man1.install 'when.1'
+    system "make", "prefix=#{prefix}", "install"
+  end
+
+  test do
+    system bin/"when", "--help"
```

Comment:
The problem is it complains like this if it can't find a preferences file at `~/.when/preferences`, and that path is [hardcoded](https://github.com/bcrowell/when/blob/master/when#L1394).